lib/uplink/ex_create_bucket_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/lib/uplink"
+)
+
+func CreateBucketExample(ctx context.Context, satelliteAddress string, apiKey string, cfg *uplink.Config, out io.Writer) (err error) {
+	errCatch := func(fn func() error) { err = errs.Combine(err, fn()) }
```

Comment:
nit: I would just write it inline.

Reply:
this is for documentation so i don't want to distract from the semantics